app/src/main/java/com/example/concordia_campus_guide/Helper/LocaleHelper.java

```diff
@@ -0,0 +1,65 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.preference.PreferenceManager;
+
+import java.util.Locale;
+
+public class LocaleHelper {
+
+    private static final String SELECTED_LANGUAGE = "Locale.Helper.Selected.Language";
+
+    public static void onCreate(Context context) {
+
+        String lang;
+        if(getLanguage(context).isEmpty()){
+            lang = getPersistedData(context, Locale.getDefault().getLanguage());
+        }else {
+            lang = getLanguage(context);
+        }
+
+        setLocale(context, lang);
+    }
+
+    public static void onCreate(Context context, String defaultLanguage) {
+        String lang = getPersistedData(context, defaultLanguage);
+        setLocale(context, lang);
+    }
```

Comment:
LocalHelper is not a fragment or an activity. Why does it have onCreate() methods ?